trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -108,6 +108,21 @@ make a decision to trace and the decision might be deferred). When `trace-option
 the default value for this bit is `0`
 * The behavior of other bits is currently undefined.
 
+# Mutating the header 
+
+Library or platform receiving `traceparent` request header MUST send it to outgoing http requests. It MAY mutate the value of this header before passing to outgoing http requests.  
+
+Library or platform receiving `traceparent` response header MAY send it as a response header to originating http request. Proxies, load balancers, and other platforms with 1:1 mapping of request and response SHOULD return this `traceparent` header to originating http request. `traceparent` response header MAY also be mutated.
+
+Here is the list of allowed mutations: 
+
+1. **Update `span-id`**. The value of property `span-id` can be regenerated. 
```

Comment:
What does this mean if done for the response?

Reply:
there are different cases. This mutation basically says you can receive the response header from somebody, but you should not preserve the `span-id` when returning `traceparent` to your caller. You can replace it with your own `span-id`.

There are also some async scenarios possible. I think we have to standardize valid mutations, not exact semantic for every scenario.